backend/app/usecase/url/urlcreator_test.go

```diff
@@ -24,6 +24,7 @@ func TestURLCreatorPersist_CreateURL(t *testing.T) {
 
 	alias := "220uFicCJj"
 	longAlias := "an-alias-cannot-be-used-to-specify-default-arguments"
+	invalidFragmentAlias := "cant-have#chr"
```

Comment:
@byliuyang As discussed in #741, we will need to update `CreateURL` to accept a string literal instead of string pointer if we want to hardcode aliases into the test cases directly